images/Makefile.common.in

```diff
@@ -30,14 +30,25 @@ GO_VERSION=$(shell cat $(CURDIR)/../../.go-version | head -n1)
 # required to enable buildx
 export DOCKER_CLI_EXPERIMENTAL=enabled
 
+ifeq ($(KIND_EXPERIMENTAL_PROVIDER),podman)
```

Comment:
`KIND_EXPERIMENTAL_PROVIDER` is meant to control kind nodes and nothing else, we should not overload this env.

We don't test or support this build with podman, there are enough quirks as is and we 100% depend on buildx specific functionality.

Reply:
NOTE: Kubernetes/Kubernetes also uses buildx and only officially supports building with docker. The images can be run with many runtimes.

Reply:
Kubernetes has a convention to allow `$DOCKER` to be overridden for trivial commands, but the lack of compatible drop-in is why this script only has docker, versus e.g. https://github.com/kubernetes-sigs/kind/blob/9966319970640e99c7240e5e216acf9dac1f5640/hack/make-rules/verify/shellcheck.sh#L32

Reply:
@BenTheElder Thanks for the pointer, I will try how far `DOCKER=podman` gets me. Making a symlink seems to mostly work but using a environment variable seems cleaner.

As for the buildx needed to build node images -- why does anything need to be build to get the Kind node image, couldn't just fetching binaries / tars from https://dl.k8s.io/ do the trick?

Reply:
> As for the buildx needed to build node images -- why does anything need to be build to get the Kind node image, couldn't just fetching binaries / tars from https://dl.k8s.io/ do the trick?

kind exists first and foremost to test kubernetes under development so that's where the design comes from
https://kind.sigs.k8s.io/docs/contributing/project-scope/#p0-support-testing-kubernetes